catalog/core/catalog-core-metacardgroomerplugin/src/main/java/ddf/catalog/plugin/groomer/metacard/StandardMetacardGroomerPlugin.java

```diff
@@ -72,6 +73,15 @@ protected void applyCreatedOperationRules(CreateRequest createRequest, Metacard
             aMetacard.setAttribute(new AttributeImpl(Metacard.TAGS,
                     Collections.singletonList(Metacard.DEFAULT_TAG)));
         }
+
+        // upon creation of the metacard, generate attributes for created, modified dates
```

Comment:
I think we should check if these dates are already set before setting them here. There are valid cases where they might already be populated.